File: src/main/java/me/geek/mail/common/Template/Sub/Temp.java
package me.geek.mail.common.Template.Sub;

import org.bukkit.inventory.ItemStack;

/**
 * 作者: 老廖
 * 时间: 2022/8/8
 **/
public interface Temp {
    String getPackID();
    String getCondition();
    String getAction();
    String getDeny();
    String getTitle();
    String getText();

    String getAppendix();
    ItemStack[] getItemAppendix();


}

File: src/main/kotlin/me/geek/mail/common/template/Sub/Temp.kt
package me.geek.mail.common.template.sub



/**
 * 作者: 老廖
 * 时间: 2022/8/8
 */
interface Temp {
    val packID: String
    val condition: String
    val action: String
    val deny: String
    val title: String
    val text: String
    val type: String
    val additional: String?
        get() = "0"
    val itemStacks: String?
        get() = ""
    val command: List<String>?
}